use/LMM-Genetic-Diversity.Rmd

```diff
@@ -0,0 +1,152 @@
+---
+title: "Linear mixed-effects models to control for the variability of microsatellite loci when comparing genetic diversity"
+---
+
+# Introduction
+
+There is general interest in comparing the amount of genetic variation among different populations, usually measured as number of alleles, heterozygosity and/or allelic richness. Petit et al. (2005) were the first to emphasize the importance of controlling for the fact that microsatellite variation is dependent from the length of the microsatellite markers employed. They proposed the use of an ANCOVA framework when comparing genetic variation of microsatellite markers, with the mean number of repeats (MNR) included as covariate.
+
+Here we suggest the use of linear mixed-effects models (LMMs) to control for the variability of microsatellite loci when comparing genetic diversity.This approach has less assumptions and is more statistically powerful than ANCOVA (Crawley 2007). 
+
+# Assumptions
+
+- The explanatory variables are related linearly to the response
+- The errors have constant variance
+- The errors are independent
+- The errors are Normally distributed
+
+# Resources
+
+## Data
+
+As an example we'll use the comparison in Allelic Richness among populations of the orchid bee Euglossa dilemma in the Yucatan peninsula (Soro et al. submitted). The 24 sites where populations were sampled were assigned to 4 categories corresponding to different degrees of habitat fragmentation: 1) Natural (N); 2) Disturbed (D); 3) City (C); 4) Island (I). We provide a data file (GenDivEuglossa.txt), where for each Locality (population) under a certain Management (fixed factor), Allelic Richness rariefied to a minimum sample size of 10 individuals(response variable), has been calculated at each Locus (random factor).The data are stored in the file called [GenDivEuglossa.txt](https://github.com/nescent/popgenInfo/blob/master/data/GenDivEuglossa.txt).
+
+## Packages
+
+Packages required:
+
+Loading the required packages:  
+
+```{r, packages, message=FALSE}
+library("lme4")
+library("MuMIn")
+library("multcomp")
+```
+
+# Analysis
+
+## Section 1: Load the data
+
+At first we need to load our dataset. We will import "GenDivEuglossa.txt". 
+
+```{r, load_data_show, eval=FALSE}
+GenDivEuglossa <- read.delim("GenDivEuglossa.txt", header = TRUE)
+summary(GenDivEuglossa)
+str(GenDivEuglossa)
+```
+
+```{r, load_data_evaluate, echo=FALSE}
+# Note: the code chunk above will be shown to the reader, but it will not run.
+# This code chunk will run, but will not be shown to the reader.
+GenDivEuglossa <- read.delim("GenDivEuglossa.txt", header = TRUE)
```

Comment:
Line 51 should be:
```r
GenDivEuglossa <- read.delim("../data/GenDivEuglossa.txt", header = TRUE)
```

This is because all of the vignettes live in the 'use' folder. When they run, they need to read the data from the 'data' folder. By typing `../data/GenDivEuglossa.txt`, we are using relative paths to say "go up one folder (../) and down into the 'data' folder (data/)".